ethereum/statetransition/build.gradle

```diff
@@ -25,7 +25,11 @@ dependencies {
   implementation project(':storage')
   implementation project(':storage:api')
 
+  implementation testFixtures(project(':infrastructure:metrics'))
```

Comment:
please remove all new implementations, we will avoid exposing anything outside of `infrastructure:metrics` instead